app/components/ui/Navigation/DropdownMenu.module.scss

```diff
@@ -92,11 +90,12 @@
 
 .categoryMenuContainer {
   position: relative;
-  min-width: 160px;
+
 
   .navigationSubMenu {
-    top: 50px;
-    left: 0;
+    @include r_max($break-tablet-s) {
+      width: calc(100vw - $general-spacing-xs);
+    }
```

Comment:
**note:** This expands the top level category dropdown full width. Forgot to implement it before.

Reply:
Just wondering - Is there a reason for choosing general-spacing-xs? Or for adding that only on the category dropdown rather the nav dropdown too?

Reply:
Only that I discovered this variable defined the correct spacing.